### ISSUE TITLE
Lose logging in "docker logs -f" if server crashes.

### ISSUE THREAD

**Issue by alatnet** [vinanrra/Docker-Satisfactory#7]
If, for ever reason (i.e. create a new save or load a new save, etc.) the server crashes, the logging output that is present in either "docker logs -f" or portainer will not update at all any more.

**Comment by vinanrra** [comment]
I will check, just notice I had the same problem.

**Comment by vinanrra** [comment]
@alatnet Fixed last commit [1534b63](https://github.com/vinanrra/Docker-Satisfactory/commit/1534b63b594e15d4879c3598ce89d523118528eb)

**Comment by vinanrra** [comment]
Proof:

```
Shutdown handler: cleanup.
Assertion failed: !HasCommands() [File:Runtime/RHI/Public/RHICommandList.h] [Line: 3816] 
Pure virtual function called!
Signal 6 caught.
tail: '/home/sfserver/log/console/sfserver-console.log' has become inaccessible: No such file or directory
tail: '/home/sfserver/log/console/sfserver-console.log' has appeared;  following new file
4.26.1-173879+++FactoryGame+rel-main-0.5.0 522 3
Disabling core dumps.
Project file not found: /home/sfserver/serverfiles/FactoryGame/FactoryGame.uproject
sh: 1: xdg-user-dir: not found
LogInit: Display: Running engine for game: FactoryGame
LogInit: Display: Project file not found: /home/sfserver/serverfiles/FactoryGame/FactoryGame.uproject
LogInit: Display: 	Attempting to find via project info helper.
LogUProjectInfo: Found projects:
LogPakFile: Display: Found Pak file ../../../FactoryGame/Content/Paks/FactoryGame-LinuxServer.pak attempting to mount.
LogPakFile: Display: Mounting pak file ../../../FactoryGame/Content/Paks/FactoryGame-LinuxServer.pak.
LogPakFile: PakFile PrimaryIndexSize=591618
LogPakFile: PakFile PathHashIndexSize=675008
```
